src/ResourceManager/EventGrid/Commands.EventGrid/help/New-AzEventGridSubscription.md

```diff
@@ -150,23 +181,23 @@ This can be webhook or eventhub
 Type: System.String
 Parameter Sets: ResourceGroupNameParameterSet, ResourceIdEventSubscriptionParameterSet, CustomTopicEventSubscriptionParameterSet
 Aliases:
-Accepted values: webhook, eventhub
+Accepted values: webhook, eventhub, storagequeue, hybridconnection
 
 Required: False
-Position: 4
+Position: Named
```

Comment:
When the positions are re-added, the documentation should be regenerated, so this change is reverted.

Reply:
fixed

Reply:
@MiYanni all comments were addressed.